<?xml version="1.0" encoding="UTF-8"?>
<testsuites tests="1" failures="1" disabled="0" errors="0" timestamp="2017-02-17T18:59:24" time="0.012" name="AllTests">
  <testsuite name="SquareCavityG20" tests="1" failures="1" disabled="0" errors="0" time="0.011">
    <testcase name="SquareCavityG20" status="run" time="0.011" classname="SquareCavityG20">
      <failure message="unknown file&#x0A;C++ exception with description &quot;&#x0A;--------------------------------------------------------&#x0A;An error occurred in line &lt;194&gt; of file &lt;/home/sarna/Desktop/DG_moments_dealii/Tests/../include_new/Base_EquationGenerator.h&gt; in function&#x0A;    void EquationGenerator::Base_EquationGenerator&lt;dim&gt;::build_triplet(std::vector&lt;Eigen::Triplet&lt;double&gt; &gt;&amp;, std::string) [with int dim = 2; std::string = std::basic_string&lt;char&gt;]&#x0A;The violated condition was: &#x0A;    in.is_open()&#x0A;Additional information: &#x0A;    Triplet cant be read, file could not be opened&#x0A;--------------------------------------------------------&#x0A;&quot; thrown in the test body." type=""><![CDATA[unknown file
C++ exception with description "
--------------------------------------------------------
An error occurred in line <194> of file </home/sarna/Desktop/DG_moments_dealii/Tests/../include_new/Base_EquationGenerator.h> in function
    void EquationGenerator::Base_EquationGenerator<dim>::build_triplet(std::vector<Eigen::Triplet<double> >&, std::string) [with int dim = 2; std::string = std::basic_string<char>]
The violated condition was: 
    in.is_open()
Additional information: 
    Triplet cant be read, file could not be opened
--------------------------------------------------------
" thrown in the test body.]]></failure>
    </testcase>
  </testsuite>
</testsuites>
